firmware/shared/src/io/io_bootloaders.c

```diff
@@ -0,0 +1,56 @@
+#include <stdint.h>
+#include "cmsis_gcc.h"
+#include "hw_utils.h"
+#include "io_bootloaders.h"
+
+extern uint32_t __boot_code_start__;
+extern uint32_t __boot_code_size__;
+
+void modifyStackPointerAndStartApp(const uint32_t *address)
```

Comment:
can you change the name of this from modifyingstackpointerandstartapp to io_bootloader_modifyingStackPointer